dirsrvtests/tests/suites/password/pwdPolicy_attribute_test.py

```diff
@@ -749,6 +749,139 @@ def fin():
 
     request.addfinalizer(fin)
 
+def test_local_tpr_maxuse_5(topology_st, create_user, request):
+    """Test TPR local policy overpass global one: passwordTPRMaxUse
+    Test that after passwordTPRMaxUse failures to bind
+    additional bind with valid password are failing with CONSTRAINT_VIOLATION
+
+    :id: c3919707-d804-445a-8754-8385b1072c42
+    :customerscenario: False
+    :setup: Standalone instance
+    :steps:
+        1. Global password policy Enable passwordMustChange
+        2. Global password policy Set passwordTPRMaxUse=5
+        3. Global password policy Set passwordMaxFailure to a higher value to not disturb the test
+        4. Local password policy Enable passwordMustChange
+        5. Local password policy Set passwordTPRMaxUse=10 (higher than global)
+        6. Bind with a wrong password 10 times and check INVALID_CREDENTIALS
+        7. Check that passwordTPRUseCount got to the limit (5)
+        8. Bind with a wrong password (CONSTRAINT_VIOLATION)
+           and check passwordTPRUseCount overpass the limit by 1 (11)
+        9. Bind with a valid password 10 times and check CONSTRAINT_VIOLATION
+           and check passwordTPRUseCount increases
+        10. Reset password policy configuration
+    :expected results:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+        8. Success
+        9. Success
+        10. Success
+    """
+
+    global_tpr_maxuse = 5
+    # Set global password policy config, passwordMaxFailure being higher than
+    # passwordTPRMaxUse so that TPR is enforced first
+    topology_st.standalone.config.replace('passwordMustChange', 'on')
+    topology_st.standalone.config.replace('passwordMaxFailure', str(global_tpr_maxuse + 20))
+    topology_st.standalone.config.replace('passwordTPRMaxUse', str(global_tpr_maxuse))
+    time.sleep(.5)
+
+    local_tpr_maxuse = global_tpr_maxuse + 5
+    # Reset user's password with a local password policy
+    # that has passwordTPRMaxUse higher than global
+    our_user = UserAccount(topology_st.standalone, TEST_USER_DN)
```

Comment:
It's better to use the fixture directly.
For that, you need to add this line to '**def create_user()**'  function:
```
return user
```
And then, in the test here, you can just call it directly like:
```
subprocess.call(['%s/dsconf' % topology_st.standalone.get_sbin_dir(),
                     'slapd-standalone1',
                     'localpwp',
                     'adduser',
                     create_user.dn])
```
Hence, it makes sense to name the fixture '**test_user**' instead of '**create_user**'.

Reply:
Changed done. Would you please have a look again. Thanks